### DISCUSSION TITLE
How can I read title on windows?

### DISCUSSION THREAD

**Discussion by Youssefbenhammouda** [dvarrazzo/py-setproctitle#105] [Q&A]
Hello,
I am trying to use this lib to give my process a title, I read in the documentary that you must use PROCESS EXPLORER in order to view the title, I tried it but I am not seeing anything.
So how can I view the title in PROCESS EXPLORER ? or maybe it's not working anymore?

**Comment by thiagobenine** [comment]
I'm facing the same problem. I would like to know how can I use it with Process Explorer.
